tmail-backend/pom.xml

```diff
@@ -41,7 +41,6 @@
         <james.groupId>org.apache.james</james.groupId>
         <james.version>3.7.0-SNAPSHOT</james.version>
         <james.protocols.groupId>${james.groupId}.protocols</james.protocols.groupId>
-        <apache-mime4j.version>0.8.5-SNAPSHOT</apache-mime4j.version>
```

Comment:
I would think it's more of a cleanup? That declared version was not used anywhere if I'm not wrong, thus the version used was the one declared by apache james project?

Reply:
No it override the versions used elsewhere.

Reply:
ah ok my bad then :)